src/main/java/io/obswebsocket/community/client/WebSocketCloseCode.java

```diff
@@ -0,0 +1,39 @@
+package io.obswebsocket.community.client;
+
+public enum WebSocketCloseCode {
+  // Internal only
+  DontClose(0),
```

Comment:
If this is internal only (i'm guessing internal only in the obs-websocket) is it necessary that we add this then?

Reply:
That is true, however I included it because z is a common enough 'default' that I think it would be wiser to include it / be transparent.

Reply:
Sure

Reply:
Agreed, let's just be as transparent as possible